src/pages/experimental/delete-pattern/index.mdx

```diff
@@ -1,94 +1,179 @@
 ---
-title: Delete pattern
-description: Deleting is a destructive action. To protect user resources, your product should support ‘undo’ where possible.
+
+title: Delete / Remove pattern
+
```

Comment:
```suggestion
description: Deleting is a destructive action. To protect user resources, your product should support ‘undo’ where possible.
```

We need to add a description back in, this is used in the search results that shows up.

Reply:
Thanks for this Alison